solenoid/settings/base.py

```diff
@@ -1,430 +1,293 @@
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
-    'django.middleware.security.SecurityMiddleware',
-    'whitenoise.middleware.WhiteNoiseMiddleware',
+    "django.middleware.security.SecurityMiddleware",
+    "django.contrib.sessions.middleware.SessionMiddleware",
+    "django.middleware.common.CommonMiddleware",
+    "django.middleware.csrf.CsrfViewMiddleware",
+    "django.contrib.auth.middleware.AuthenticationMiddleware",
+    "django.contrib.messages.middleware.MessageMiddleware",
+    "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    # 'whitenoise.middleware.WhiteNoiseMiddleware',
 ]
 
+ROOT_URLCONF = "solenoid.urls"
 
-# DEBUG
-# -----------------------------------------------------------------------------
+TEMPLATES = [
+    {
+        "BACKEND": "django.template.backends.django.DjangoTemplates",
+        "DIRS": [os.path.join(BASE_DIR, "solenoid", "templates")],
+        "APP_DIRS": True,
+        "OPTIONS": {
+            "context_processors": [
+                "django.template.context_processors.debug",
+                "django.template.context_processors.request",
+                "django.contrib.auth.context_processors.auth",
+                "django.contrib.messages.context_processors.messages",
+            ],
+        },
+    },
+]
 
-DEBUG = boolean(os.getenv('DJANGO_DEBUG', False))
 
-# DATABASE CONFIGURATION
-# -----------------------------------------------------------------------------
+WSGI_APPLICATION = "solenoid.wsgi.application"
+SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 
-# https://docs.djangoproject.com/en/1.8/ref/settings/#databases
+# SITES
+SITE_ID = 1
 
+# DATABASE
 DATABASES = {
-    'default': dj_database_url.config(
-        default=os.getenv('DATABASE_URL', 'sqlite:///db.sqlite3'),
-        conn_max_age=600)
+    "default": dj_database_url.config(
+        default=os.getenv("DATABASE_URL", "sqlite:///db.sqlite3"), conn_max_age=600
+    )
 }
 
-
-# GENERAL CONFIGURATION
-# -----------------------------------------------------------------------------
-
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.environ['DJANGO_SECRET_KEY']
-
-# This will accept a comma-separated list of allowed hosts
-ALLOWED_HOSTS = make_list(os.getenv('ALLOWED_HOSTS'))
-
-if 'HEROKU_APP_NAME' in os.environ:
-    ALLOWED_HOSTS.append(
-            '{}.herokuapp.com'.format(os.environ['HEROKU_APP_NAME']))
-
-ROOT_URLCONF = 'solenoid.urls'
-
-WSGI_APPLICATION = 'solenoid.wsgi.application'
-
-SITE_ID = 1
-
-SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
-
-# INTERNATIONALIZATION CONFIGURATION
-# -----------------------------------------------------------------------------
-
-# https://docs.djangoproject.com/en/1.8/topics/i18n/
-
-LANGUAGE_CODE = 'en-us'
-
-TIME_ZONE = 'UTC'
+# INTERNALIZATION
```

Comment:
`# INTERNATIONALIZATION`?

Reply:
Hmm, [this](https://github.com/MITLibraries/solenoid/blob/main/solenoid/settings/base.py#L126C2-L126C2) was the original comment header 🤔. I think the comments were to indicate the category of each setting: https://docs.djangoproject.com/en/4.2/topics/i18n/!

Reply:
Oooh, 😅 . I fixed it!